pkg/tsdb/interval/interval.go

```diff
@@ -67,6 +67,11 @@ func (ic *intervalCalculator) Calculate(timerange plugins.DataTimeRange, minInte
 }
 
 func GetIntervalFrom(dsInfo *models.DataSource, queryModel *simplejson.Json, defaultInterval time.Duration) (time.Duration, error) {
+	intervalMs := queryModel.Get("intervalMs").MustInt64(-1)
+	if intervalMs != -1 {
+		return time.Millisecond * time.Duration(intervalMs), nil
```

Comment:
Just a nit, the standard in Go is [consistent case for initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms):
```suggestion
	intervalMS := queryModel.Get("intervalMs").MustInt64(-1)
	if intervalMS != -1 {
		return time.Millisecond * time.Duration(intervalMS), nil
```